package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "smart-home-testing",
+  "version": "1.0.0",
+  "description": "Testing library for Smart Home Actions for developers to create and run integration tests locally. ",
+  "main": "src/validate.ts",
+  "scripts": {
+    "test": "mocha",
+    "compile": "tsc",
+    "check": "gts check",
+    "clean": "gts clean",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/googleinterns/smart-home-testing.git"
+  },
+  "author": "Jolene Lozano",
```

Comment:
@Fleker Not sure on project rules, should this be Google or the team name or the individual author?

Reply:
Formally it should be "Google LLC".